web/pgadmin/preferences/static/js/components/PreferencesComponent.jsx

```diff
@@ -689,7 +690,7 @@ export default function PreferencesComponent({ ...props }) {
     const text = `${gettext('All preferences will be reset to their default values.')}<br><br>${gettext('Do you want to proceed?')}<br><br>
 ${gettext('Note:')}<br> <ul style="padding-left:20px"><li style="list-style-type:disc">${gettext('The object explorer tree will be refreshed automatically to reflect the changes.')}</li>
 <li style="list-style-type:disc">${gettext('If the application language changes, a reload of the application will be required. You can choose to reload later at your convenience.')}</li></ul>`;
-
+    const modalId = MODAL_DIALOGS.RESET_PREFERNCES;
```

Comment:
why do we need a var if we can directly pass it?

Reply:
Done. I thought it to have single place, hence was added like this.

Reply:
missed this place?

Reply:
Done.